### PR TITLE
Add mix format

### PR DESCRIPTION
**Pull request by jeffjewiss**
Thought you might be interested in using `mix format` on this project. Cheers! 👋

### CODE REVIEW

**Comment by sobolevn** [comment]
Maybe we should drop some old `elixir` versions?

**Comment by samueljseay** [comment]
@sobolevn yeah sounds good. Sorry I haven't had much time for this project lately. I don't have the  fortune of getting to work with/on Elixir things at the moment. I'd be happy if someone else could pick up the work to get this PR passing as I'm totally happy if we merge this.

**Comment by jeffjewiss** [comment]
This should be good to go now if you're happy with the chosen Elixir versions.

File: mix.exs [hunk 1]
```diff
@@ -9,25 +9,24 @@ defmodule Recaptcha.Mixfile do
       description: description(),
       deps: deps(),
       package: package(),
-
-      build_embedded: Mix.env == :prod,
-      start_permanent: Mix.env == :prod,
+      build_embedded: Mix.env() == :prod,
```
Comment (sobolevn) [inline mix.exs]: Can you please update `:elixir_version` value as well? 
As we do not run tests in `1.2` anymore.

Comment (jeffjewiss) [reply mix.exs]: Yeah, no problem. I'll update this to 1.4

**Comment by sobolevn** [comment]
Thanks!

**Pull request merged**